### DISCUSSION TITLE
Request to filter Review List comment actions using apply_filters('comment_row_actions)

### DISCUSSION THREAD

**Discussion by sackadelic** [woocommerce/woocommerce#52461] [General]
I'm working on a plugin that can feature / unfeature WooCommerce reviews. I'd like to be able to filter the `comment_row_actions` on the Products > Reviews table:

![CleanShot 2024-10-30 at 09 56 07@2x](https://github.com/user-attachments/assets/5de4f810-b5e9-4ff3-b48d-1c10e7ec34c9)

It looks like adding:  `$actions = apply_filters( 'comment_row_actions', array_filter( $actions ), $item );` to the ReviewsListTable.php would work?

**Comment by barryhughes** [comment]
We've converted this into an enhancement request :-)

https://github.com/woocommerce/woocommerce/issues/52679

We'd also be receptive to a pull request, if you wished to submit one.
